backend/event/models.py

```diff
@@ -148,13 +152,38 @@ def __str__(self):
 class LetterEventDate(Field, LettercraftDate, models.Model):
 
     letter_action = models.OneToOneField(
-        to=LetterAction, on_delete=models.CASCADE, related_name="date"
+        to=LetterActionBase, on_delete=models.CASCADE, related_name="date"
     )
 
     def __str__(self):
         return f"{self.letter_action} ({self.display_date})"
 
 
+class LetterAction(Historical, LetterActionBase, models.Model):
+    """
+    An aggregate model that represents a letter action in history. This model is based on one or multiple SourceDescriptions and other sources that are not part of this database.
+    """
+
+    pass
+
+
+class LetterActionDescription(SourceDescription, LetterActionBase, models.Model):
+    """
+    A description of a letter action in a source.
+    """
```

Comment:
I'm not sure if there is any need for the "historical" letter action. In the case of agents, it makes sense that a description of an individual in a source will often correspond to a known historical figure, and that these historical figures are a useful "entry point" into the database. Cross-referencing individual actions is much harder, if ever possible, and less useful for making queries.

I would argue that while a `SourceDescription` is often referring to something "real" (e.g. a person, an action, a place), it's not always useful to represent both layers in the database, if they don't serve the analysis. (In the case of actions, it is also tricky that actions in texts do not necessarily represent "real" events but may serve a literary purpose.)